test/t/unit/test_unit_compgen_filedir.py

```diff
@@ -228,7 +228,21 @@ def test_26(self, bash, functions, funcname):
         completion = assert_complete(
             bash, r'%s "a\$b/' % funcname, cwd="_filedir"
         )
-        assert completion == 'h"'
+        try:
+            assert completion == 'h"'
+        except AssertionError:
+            bash_version = assert_bash_exec(
+                bash,
+                r'printf "%s.%s\n" "${BASH_VERSINFO[0]}" "${BASH_VERSINFO[1]}"',
+                want_output=True,
+            ).strip()
+            # This is workaround for https://github.com/scop/bash-completion/issues/1435, see
+            # https://lists.gnu.org/archive/html/bug-bash/2025-08/msg00107.html
```

Comment:
After that, I posted a follow-up email to bug-bash, but Chet's reply [1] was that the above page (`2025-08/msg00107.html`) is similar, but a different issue (though I'm not fully convinced). I think it would be better to link [1] rather than the above one.

[1] https://lists.gnu.org/archive/html/bug-bash/2025-09/msg00332.html

Reply:
Thanks, Updated